dataland-backend/src/main/kotlin/org/dataland/datalandbackend/validator/DataPointMaximumValue.kt

```diff
@@ -0,0 +1,50 @@
+package org.dataland.datalandbackend.validator
+
+import jakarta.validation.Constraint
+import jakarta.validation.ConstraintValidator
+import jakarta.validation.ConstraintValidatorContext
+import jakarta.validation.Payload
+import org.dataland.datalandbackend.interfaces.datapoints.BaseDataPoint
+import org.dataland.datalandbackendutils.exceptions.InvalidInputApiException
+import java.math.BigDecimal
+import kotlin.properties.Delegates
+import kotlin.reflect.KClass
+
+/**
+ * Annotation for the validation of an ExtendedDataPoint<*> holding a number
+ */
+@Target(AnnotationTarget.FIELD)
+@Constraint(validatedBy = [DataPointMaximumValidator::class])
+annotation class DataPointMaximumValue(
+    val maximumValue: Long = 100,
```

Comment:
Why is the maximumValue set to 100?

Reply:
default max is 100, because it is mostly used for percentages. However, the default is always manually overridden for readability, so this is (st the moment) not necessary

Reply:
ok, so we keep it as a safety net